Automattic/wp-calypso
# Search: match Jetpack Cloud look and feel

#### Changes proposed in this Pull Request

The Jetpack Search section was originally only shown in Calypso blue, so exclusively uses styles that look like regular Calypso. This looks out-of-place when rendered in Jetpack Cloud.

This PR updates the section to use the Jetpack Upsell component in Cloud, which matches the style of the Backup and Scan sections. We have preserved the existing look and feel in Calypso blue.

#### Testing instructions

1. Run locally with `CALYPSO_ENV=jetpack-cloud-development yarn start` and map jetpack.cloud.localhost to 127.0.0.1 in your “hosts” file.
2. Visit http://jetpack.cloud.localhost:3000/ and choose the 'Search' item in the sidebar.
3. Ensure the appearance of the section matches other Jetpack sections like Backup and Scan (no container box, correct fonts and sizes).

<img width="734" alt="Screen Shot 2021-06-02 at 12 01 11" src="https://user-images.githubusercontent.com/17325/120405540-e1783c80-c39c-11eb-9b37-745c9b99dfc4.png">

4. Stop Calypso and restart with the regular environment (yarn start) and access via http://calypso.localhost:3000.
5. Ensure the appearance of the section matches other Jetpack sections like Backup and Scan (container box, correct fonts and sizes).

<img width="779" alt="Screen Shot 2021-06-02 at 16 43 07" src="https://user-images.githubusercontent.com/17325/120425043-a4727100-c3c1-11eb-9922-b1f8926b6a05.png">
